addons/file-sync/Commands/Import.ts

```diff
@@ -100,10 +101,10 @@ export default class Import extends BaseCommand {
       fileSyncOptionsToProcess.push(fileSyncOption);
     } else {
       this.logger.debug('no file sync option id found');
-      const fileSyncOptions = await (
-        await this.deps.FileSyncOption.find({ enabled: true })
-      ).all();
-      this.logger.debug(`pushing ${fileSyncOptions.length} to process`);
+      const fileSyncOptions = await this.deps.FileSyncOption.query({
+        enabled: true,
+      }).all();
+      this.logger.debug(`pushing ${fileSyncOptions?.length} to process`);
```

Comment:
```suggestion
      this.logger.debug(`pushing ${fileSyncOptions.length} to process`);
```

If the value can be something other than an array, it's a bug in adonis-mongodb